examples/webhook/README.md

```diff
@@ -0,0 +1,130 @@
+# Datagen and Materialize Webhooks Source
+
+### What Are Webhooks?
+
+Webhooks are a method for augmenting or altering the behavior of a web page, or web application, with custom callbacks. These callbacks may be maintained, modified, and managed by third-party users and developers who may not necessarily be affiliated with the originating website or application.
```

Comment:
This explanation feels too formal and vague to me. I think we can say something simpler, like “a webhook is a service that sends HTTP POST requests to a target API endpoint when an event occurs. Webhooks are a great way for services to proactively share events when they occur. For example, GitHub allows users to create webhooks that send an event to an endpoint of their choice whenever, say, an issue is filed on their GitHub project.”